rules/cross-platform/persistence_credential_access_modify_auth_module_or_config.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
+creation_date = "2020/12/21"
+maturity = "production"
+updated_date = "2020/12/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may modify the standard authentication module for persistence via patching the normal authorization process
+or modifying the login configuration to allow unauthorized access or elevate privileges.
+"""
+false_positives = ["Trusted system module updates or allowed Pluggable Authentication Module (PAM) daemon configuration changes."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Modification of Standard Authentication Module or Configuration"
+references = [
+    "https://github.com/zephrax/linux-pam-backdoor",
+    "https://github.com/eurialo/pambd",
+    "http://0x90909090.blogspot.com/2016/06/creating-backdoor-in-pam-in-5-line-of.html",
+    "https://www.trendmicro.com/en_us/research/19/i/skidmap-linux-malware-uses-rootkit-capabilities-to-hide-cryptocurrency-mining-payload.html",
+]
+risk_score = 47
+rule_id = "93f47b6f-5728-4004-ba00-625083b3dcb0"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Linux", "Threat Detection", "Credential Access", "Persistence"]
+type = "query"
+
+query = '''
+event.category:file and event.type:change and
+process.name:* and 
```

Comment:
@rw-access @brokensound77 - Is our rule of thumb to avoid leading wildcards like this with no other characters? My concern is that this rule might consume considerable resources.

Reply:
in KQL, this `: *` means the field "exists". The syntax is a little overloaded, but this is a special case.

@Samirbous I think... this should be `process.executable`, since you do a `not` check on that field.

Reply:
Today I Learned. Thanks 👍